client/src/org/apache/cloudstack/ServerDaemon.java

```diff
@@ -129,6 +126,7 @@ public void init(final DaemonContext context) {
             setKeystorePassword(properties.getProperty(KEYSTORE_PASSWORD));
             setWebAppLocation(properties.getProperty(WEBAPP_DIR));
             setAccessLogFile(properties.getProperty(ACCESS_LOG, "access.log"));
+            setSessionTimeout(Integer.valueOf(properties.getProperty(SESSION_TIMEOUT, "10")));
```

Comment:
@marcaurele you have a default value in the property file, which is 30, and here it is 10 (at line 85). Moreover, what about using the value of variable `sessionTimeout ` as the default values? The variable already starts with the default value anyways.

Reply:
Thanks, I've synced default values to 30 now.